shlr/meson.build

```diff
@@ -24,63 +30,68 @@ if not capstone_dep.found() or not get_option('use_sys_capstone')
 
     message('Patching capstone with radare\'s patches')
     foreach file : patches_files
-      patch_cmd = 'patch -p1 -d capstone -i ../capstone-patches/@0@'.format(file)
+      patch_cmd = 'patch -p1 -d @0@ -i ../capstone-patches/@1@'.format(capstone_path, file)
```

Comment:
Ok but I have again one question. Why appveyor is green then? If we shouldn't allow these tools to be there, we should remove them from the testing environment too, otherwise it's hard to know what is available on windows and what is not.

Do you have an alternative on windows for `patch` or should I just avoid the entire patch code?

Reply:
AppVeyor is green because `run_command` doesn't check return code. http://mesonbuild.com/Reference-manual.html#run_command

Reply:
I think `git-am` can be used to do the same, no?